contrib/verify-commits/README.md

```diff
@@ -33,6 +33,17 @@ Configuration files
 * `trusted-keys`: This file should contain a \n-delimited list of all PGP fingerprints of authorized commit signers (primary, not subkeys).
 * `allow-revsig-commits`: This file should contain a \n-delimited list of git commit hashes. See next section for more info.
 
+Import trusted keys
+-------------------
+In order to check the commit signatures you must add the trusted PGP keys to your machine. This can be done in Linux by running
+    while read -r LINE; do gpg --recv-keys $LINE; done < contrib/verify-commits/trusted-keys
```

Comment:
This is a markdown file. I suggest you surround this line with backticks to show that it's a command to type, ie:

In order to check the commit signatures you must add the trusted PGP keys to your machine. This can be done in Linux by running:
```
while read -r LINE; do gpg --recv-keys $LINE; done < contrib/verify-commits/trusted-keys
```

This also assumes that the current working directory is the repo's root directoy. I'd assume it to be the verify-commits directory (ie the command would be `while read -r LINE; do gpg --recv-keys $LINE; done < trusted-keys`)

Reply:
Bonus points: add `sh` as hint for syntax highlighting
```
```sh
while read...
```

@jnewbery:
> assume it to be the verify-commits directory

Agreed, but then the main command should probably also assume that. I wonder if that causes problems if it reaches old commits where this directory doesn't exist?

Reply:
Changed to use backticks with the shell suggestion.

I'm ambivalent about the root directory assumption; the reason I went with this assumption was due to the command that already exists in the readme that makes this same assumption.

Reply:
> I'm ambivalent about the root directory assumption

Yep, I'm not really too fussed either way.